Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityToolkitFiles.cs

```diff
@@ -101,10 +101,9 @@ public static void OnPostprocessAllAssets(string[] importedAssets, string[] dele
         private static Task searchForFoldersTask = null;
         private static CancellationTokenSource searchForFoldersToken;
 
-        private static string NormalizeSeparators(string path) => 
-            path?.Replace('\\', Path.DirectorySeparatorChar).Replace('/', Path.DirectorySeparatorChar);
-
-        private static string FormatSeparatorsForUnity(string path) => path?.Replace('\\', '/');
+        // This ensures directory separator chars are platform independent. Given path might use \ or /
+        private static string NormalizeSeparators(string path) 
+            => path?.Replace('\\', Path.DirectorySeparatorChar).Replace('/', Path.DirectorySeparatorChar);
```

Comment:
Is it possible to use the NormalizeSeparators string extension here instead of redefining?

Reply:
I hope...it wasn't recognizing when I initially tried. Need to figure out what's going on

Reply:
This is because of #7152